src/graph/parser.ts

```diff
@@ -135,6 +137,25 @@ export class Graph {
         }
     }
 
+    private static parseSize(cssString: string):{value: string, isDynamic: boolean} {
+        cssString = cssString.toString()
+        let index = cssString.search(/[A-Za-z%]/)
+        let number
+        
+        if (number && number > MAX_SIZE) {
+            throw new SyntaxError(`Graph size outside of accepted bounds (must be <${MAX_SIZE}x${MAX_SIZE})`);
+        }
+        let unit
+        if (index != -1) {
+            number = parseFloat(cssString.substring(0, index))
+            unit = cssString.substring(index)
+        } else {
+            number = parseFloat(cssString)
+            unit = "px"
+        }
+        return { value: number.toString() + unit, isDynamic: unit == "%" }
+    }
+
```

Comment:
It would make more sense if this was just inlined with the switch case - as the rest of the parsers are.